db_books.rb

```diff
@@ -0,0 +1,29 @@
+require 'json'
+
+module BooksPersistence
+  def store_books(books)
+    data = []
+    file = './books.json'
+    !(File.exist?(file) && File.read(file) != '') && File.write(file, '[]')
+
+    return unless File.exist?(file)
+
+    books.each do |book|
+      data << { title: book.title, author: book.author }
+    end
+    File.write(file, JSON.generate(data))
+  end
+
+  def load_books
+    data = []
+    file = './books.json'
+    !(File.exist?(file) && File.read(file) != '') && File.write(file, '[]')
+    return data unless File.exist?(file) && File.read(file) != ''
+
+    JSON.parse(File.read(file)).each do |book|
+      data << Book.new(book['title'], book['author'], book['id'])
```

Comment:
- [ ] Kindly be noticed that the `Book` class takes only 2 parameters (title, author) and you are sending to it 3 parameters (title, author, id), and this is causing an error please fix that.